src/main/resources/org/jenkinsci/plugins/vSphereCloudSlaveTemplate/help-cloneNamePrefix.html

```diff
@@ -1,6 +1,6 @@
 <div>
 Controls the name by which the virtual machine will be known to vSphere.
-The plug-in will append an underscore and a unique ID to this prefix in order to generate unique names.<p>
+The plug-in will append a unique ID to this prefix in order to generate unique names.<p>

```

Comment:
HTML nit (maybe applies throughout) -- do this

```
<p>stuff</p>

<p>more stuff</p>
```

Reply:
Personally, I'm not much of a fan of putting formatting/layout elements at the start/end of lines that contain content.  I prefer to keep layout and content on their own lines. e.g. 
```html
<p>
stuff
</p>
<p>
more stuff
</p>
```

However, I did try wrapping the first line of text (line 2 in the file) in `<p>` ... `</p>` and found that this then added an unsightly amount of vertical whitespace in the help text (between the top border of the help box and the top of the first line of text).
There's a border around the existing help-text that's about as wide & tall as an "M" character, and the word "Controls" at the start and the "(from vSphere Plugin)" text at the end are both indented by equal amounts.
If we make the initial text a paragraph then the layout becomes unbalanced and there's "two Ms" worth of vertical whitespace before the first word "Controls".
i.e. it might be "more correct" HTML but it's also more ugly.

I suspect that the HTML generated by Jenkins to wrap the help text was designed more for "simple text" than HTML containing lots of layout formatting.  I suspect that it was designed to make things "look right" if they start and end with no vertical whitespace.

I'm not an HTML expert, so I did `mvn hpi:run` and did some experimentation with the layout ... and I found that if we remove `<p>` and `</p>` completely and instead separate the initial text (lines 2 and 3 in this file) from the "Note" (line 4) by `<br><br>` then that looks good too (in my opinion), and it also uses half an M less vertical whitespace between the "Note" text and the "(from vSphere Plugin)" text, which appeals to my sense of neatness.

...also (a brief google later), doing `<p style="margin-top:0;">stuff</p>` for the first makes it look acceptable when rendered too, but I suspect that messing with styling here would be considered worse than just using `<br><br>` would be.

I'm open to ideas but unless we can settle on something that's both "good HTML practice" and also "looks correct when rendered" then I think it may be best to leave that code as-is.